src/wolfscp.c

```diff
@@ -1110,16 +1118,15 @@ static int ScpCheckForRename(WOLFSSH* ssh, int cmdSz)
     return WS_SUCCESS;
 }
 
+#endif
 
 /* helps with checking if the base path is a directory or file
  * returns WS_SUCCESS on success */
 static int ParseBasePathHelper(WOLFSSH* ssh, int cmdSz)
 {
-    int ret;
-
-    ret = ScpCheckForRename(ssh, cmdSz);
-
+    int ret = 0;
 #ifndef NO_FILESYSTEM
```

Comment:
Why can the file in the cmd not be renamed in the case that there is NO_FILESYSTEM?

Reply:
Indeed `ScpCheckForRename` can be called outside the `#ifndef ...` block. It should be fixed now.